src/app/applyContextMenu.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Menu, MenuItem } from 'electron'
+
+/**
+ * Enable context menu by right click
+ *
+ * @param {import('electron').BrowserWindow} browserWindow - Browser window
+ */
+export function applyContextMenu(browserWindow) {
+
+	browserWindow.webContents.on('context-menu', async (event, params) => {
+		event.preventDefault()
+		// browserWindow.webContents.send('open-context-menu', params)
+		const menu = new Menu()
+
+		// Add each spelling suggestion
+		for (const suggestion of params.dictionarySuggestions) {
+			menu.append(new MenuItem({
+				label: suggestion,
+				click: () => browserWindow.webContents.replaceMisspelling(suggestion),
+			}))
+		}
+		// Allow users to add the misspelled word to the dictionary
+		if (params.misspelledWord) {
+			menu.append(
+				new MenuItem({
+					label: 'Add to dictionary',
+					click: () => browserWindow.webContents.session.addWordToSpellCheckerDictionary(params.misspelledWord),
+				}),
+			)
+		}
+
+		if (params.selectionText && params.editFlags.canCut) {
+			menu.append(
+				new MenuItem({
+					label: 'Cut selected',
+					role: 'cut',
+				}),
+			)
+		}
+		if (params.selectionText && params.editFlags.canCopy) {
+			menu.append(
+				new MenuItem({
+					label: 'Copy selected',
+					role: 'copy',
+				}),
+			)
+		}
+		if (params.editFlags.canPaste) {
+			menu.append(
+				new MenuItem({
+					label: 'Paste',
+					role: 'paste',
+				}),
+			)
+		}
```

Comment:
For MenuItems with standard roles, labels should be predefined by Electron. It also includes l10n.